src/pipdeptree/_models/package.py

```diff
@@ -83,22 +90,11 @@ def as_frozen_repr(obj: DistInfoDistribution) -> str:
         # pip._vendor.pkg_resources.DistInfoDistribution.
         #
         # This is a hacky backward compatible (with older versions of pip) fix.
```

Comment:
With https://github.com/tox-dev/pipdeptree/pull/336, we now expect the pip we are using to be >= 23.1.2. So we can remove these comments as well as it had to do with dealing with `DistInfoDistribution` objects.

Reply:
Done